vignettes/plotting_vignette.Rmd

```diff
@@ -0,0 +1,160 @@
+---
+title: "Plotting Functionality in tcplfit2"
+author: "Center for Computational Toxicology and Exposure"
+output:
+   prettydoc::html_pretty:
+    theme: architect
+    toc: yes
+    toc_depth: 4
+vignette: >
+  %\VignetteIndexEntry{Plot_with_tcplfit2}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, packages, warning=FALSE, message=FALSE}
+library(tcplfit2)
+library(dplyr)
+library(stringr)
+library(ggplot2)
+library(gridExtra)
+```
+
+```{r, include=FALSE}
```

Comment:
Not sure why this code chunk is included here? It should not be necessary once the functions are included in the package, correct?